charts/opentelemetry-operator/templates/clusterrolebinding.yaml

```diff
@@ -20,7 +21,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   labels:
-    app.kubernetes.io/name: opentelemetry-operator
```

Comment:
Does this label need completely removed?  Can it be updated to properly handle name override?

Reply:
Thanks @ngnix, I thought I was looking at an example render and it had been removed from the output.